### PR TITLE
Added welsh font characters

### LINKED ISSUE #769: Some characters in Welsh are not rendered in the standard font
### Expected behaviour
All welsh text should render correctly using the standard EQ font.

### Actual behaviour
Characters like ŷ are rendered in a different font and appear bold.

### Steps to reproduce the behaviour
Load the Welsh language version of either the Census Household or Individual survey and navigate to a question containing a ŷ character.

#### Browser
Safari

#### Operating System
OSX

### PR DESCRIPTION
**Pull request by bennpearson**
### What is the context of this PR?
Fixes #769 

### How to review 
Check fonts are compiled and loaded on in the survey

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -125,7 +131,8 @@ gulp.task('compile', [
   'bundle:scripts',
   'copy:scripts',
   'build:styles',
-  'build:images'
+  'build:images',
+  'build:fonts'
```
Comment (hamishtaplin) [inline gulpfile.babel.js:135]: Add this to watch task too

**Review by hamishtaplin — CHANGES_REQUESTED** [review]

**Review by hamishtaplin — APPROVED** [review]

**Comment by alexmorris** [comment]
LGTM

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
I just tried this and there is a noticeable delay between the screen being rendered and the text being rendered.

**Comment by hamishtaplin** [comment]
@ajmaddaford There's not a great deal we can do about that for now, it's fairly standard and only occurs once.

**Comment by ajmaddaford** [comment]
@hamishtaplin I was getting the delay on every page

**Comment by alexmorris** [comment]
@ajmaddaford we need to look into how we have caching set up (or not as the case may be?) for static assets like these.

**Comment by ajmaddaford** [comment]
I've raised a PR on the ONS assets CDN repo to add the Lato font - https://github.com/ONSdigital/cdn.ons.gov.uk-assets/pull/3

**Comment by ajmaddaford** [comment]
The font has been added to the CDN, but cache headers aren't being set (it just uses Etags). A story has been added to the digital publishing teams backlog to enable cache headers and should be done some time in the next 2 weeks.

**Comment by ajmaddaford** [comment]
Happy for this to go in while we wait for the CDN to be updated to add the cache headers (as Apache will cache the fonts in pre-prod and production).

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**